src/services/search/autoAlarm.ts

```diff
@@ -6,25 +6,26 @@ import {
   deletePath,
   postData,
 } from "../../base"
+import toQueryString from "../../lib/toQueryString"
 
 import { Paginated } from "../../types/pagination"
-import { DealerSavedSearch, AutoAlarmType } from "../../types/models/autoAlarm"
+import { DealerSavedSearch } from "../../types/models/autoAlarm"
 import { WithValidationError } from "../../types/withValidationError"
 import { withTokenRefresh } from "../../tokenRefresh"
 
 export const fetchSavedSearches = async (
   dealerId: number,
-  type: AutoAlarmType,
   page?: number,
   size?: number
 ): Promise<Paginated<DealerSavedSearch>> => {
   return withTokenRefresh(async () => {
+    const query = toQueryString({ page, size })
     try {
       const result = await fetchPath(
         Service.SEARCH,
-        `dealers/${dealerId}/listing-saved-searches/auto-alarms?type=${type}${
-          page ? `&page=${page}` : ""
-        }${size ? `&page=${size}` : ""}`
+        `dealers/${dealerId}/listing-saved-searches/auto-alarms${
+          query ? `?${query}` : ""
+        }`
```

Comment:
same here :)

Reply:
What would be the right thing to do here? Do I need to create a PR to remove the type and another 1 to fix the bug?

Reply:
if they're independent of each other, that's what I would do. if you can only roll out the type when fixing the bug, I'd rewrite git history to contain two commits - one to `fix` the bug, one for the `feat` and then instead of squashing, do a rebase merge